test/cli/test_log_runtime_archive.rb

```diff
@@ -510,6 +510,102 @@ def should_archive_dataset(dataset, archive_basename, full:)
                 end
             end
 
+            describe "#ensure_free_space" do
```

Comment:
I dont get what are the "files" supposed to be. Are they temporary files created in memory?

Reply:
> Are they temporary files created in memory?

yes, the `@archive_dir = make_tmppath` in line 515 makes a temporary directory that is removed on test teardown
![image](https://github.com/rock-core/tools-syskit/assets/79292544/34fcb213-c556-4e54-b3d0-db126e483def)

Reply:
Yeah, but thats a directory, are you mocking real files inside this directory? I couldnt figure out while reading the tests

Reply:
Just now I saw that you do write the files in the tmp_dir, got it.